css-grid-layout/esra/styles.css

```diff
@@ -10,18 +12,41 @@ body {
 
 .header {
   background-color: #006c67;
+  grid-column: 1/ span 3;
+ 
 }
 .sidebar-left {
   background-color: #003844;
+  grid-row: 2 / -1;
+  
+  
 }
 .sidebar-right {
   background-color: #ffb100;
+  
+ }
+
+@media only screen and (max-width: 900px) {
+  .sidebar-left, .sidebar-right {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .sidebar-left, .sidebar-right {
+    visibility: hidden;
```

Comment:
It should be 
```suggestion
    dispaly: none; 
```

Reply:
Apart from hiding sidebars, you want to change the grid direction so that you have one column with multimple raws on smaller dispaly. Right now Content section and footer are displayed side by side on small screens.